jobs/standalone/templates/config/redis.conf

```diff
@@ -7,13 +7,40 @@ appendonly yes
 # we should already be *in* the store dir
 appendfilename "redis.aof"
 appendfsync everysec
+
+<% if p('redis.no-appendfsync-on-rewrite') %>
+no-appendfsync-on-rewrite "<%= p('redis.no-appendfsync-on-rewrite') %>"
+<% end %>
+<% if p('redis.auto-aof-rewrite-percentage') %>
+auto-aof-rewrite-percentage "<%= p('redis.auto-aof-rewrite-percentage') %>"
+<% end %>
+<% if p('redis.auto-aof-rewrite-min-size') %>
+auto-aof-rewrite-min-size "<%= p('redis.auto-aof-rewrite-min-size') %>"
+<% end %>
+<% end %>
+
+<% if not p('lua.scripting-enabled') %>
+rename-command EVAL ""
+rename-command EVALSHA ""
+<% else %>
+# lua scripting enabled
 <% end %>
 
 maxmemory-policy "<%= p('redis.maxmemory-policy') %>"
+maxmemory <%= p('redis.maxmemory') %>
+
 notify-keyspace-events "<%= p('redis.notify-keyspace-events') %>"
 slowlog-log-slower-than <%= p('redis.slowlog-log-slower-than') %>
 slowlog-max-len <%= p('redis.slowlog-max-len') %> 
 
 timeout <%= p('client.timeout') %>
 tcp-keepalive <%= p('client.tcpkeepalive') %>
 maxclients  <%= p('client.connections') %>
+
+# Hard coding out these values per LM request
+
```

Comment:
drop this comment. It's not really by request. This is a reasonable boundary for user of a service vs operator of a service